source/assets/locales/es.json

```diff
@@ -315,12 +315,14 @@
     "addNewChain": "Agregar Nueva Cadena",
     "viewTheAddresses": "Ver las direcciones de tus cuentas permitidas",
     "errorCreatingHardWallet": "Error al crear monedero hardware.",
-    "hardwareWallet": "MONEDERO HARDWARE",
-    "selectTheHardware": "Selecciona el monedero hardware que desees",
+    "hardwareWallet": "CONECTAR HARDWALLET",
+    "selectTheHardware": "SELECT HARDWALLET:",
```

Comment:
Wrong text on line 319 of source/assets/locales/es.json
Where it says: SELECT HARDWALLET
It should say: SELECCIONAR HARDWALLET

Reply:
Nice catch, thanks!